### ISSUE TITLE
Labware Library: Tube low-level details in labware card and details page

### ISSUE THREAD

**Issue by mcous** [Opentrons/opentrons#3447]
## overview

As a labware library user, I would like to see relevant tube information for Opentrons tube racks and aluminum blocks in the labware card and details page.

Blocked by #3241, related to #3245 

## acceptance criteria

- [x] Tube displayName displayed in card (e.g. "Falcon Conical Tube x 6")
- [x] Tube brand name displayed in card title
- [x] Tube SKUs shown in labware details page
- [x] Link to tube manufacture(s) in details page
- [x] Tube measurements shown in details page

**Comment by mcous** [comment]
Closing in favor of #3477
